tools/pr-docs

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+if [ "$#" -ne 2 ] ; then
+    echo "Usage: $0 pr-number output/location" >&2
+    echo "Render documentation from a pull request" >&2
+    exit 1
+fi
+
+pr=$1
+outdir=$2
+
+# Linux || OSX
+tmpdir=$(mktemp -d --tmpdir 2>/dev/null || mkdir -d -t spectre-doc)
+trap 'rm -rf "${tmpdir}"' EXIT
+
+spectre=$(git rev-parse --show-toplevel)
```

Comment:
This script assumes it's run from inside the source directory and I get the error `fatal: not a git repository (or any of the parent directories): .git` which may not be helpful. Is there a way to catch this error and print a message explaining the issue to the user?